forgerock-core/src/main/java/org/forgerock/android/auth/biometric/DeviceCredentialFragment.java

```diff
@@ -74,12 +73,12 @@ public void onActivityResult(int requestCode, int resultCode, @Nullable Intent d
         }
         if (requestCode == LOCK_REQUEST_CODE) {
             if (resultCode == RESULT_OK) {
-                this.biometricAuth.getBiometricAuthListener().onSuccess(null);
+                this.biometricAuth.getBiometricAuthListener().onAuthenticationSucceeded(null);
             } else {
                 Logger.debug(TAG, "Fail to approve using device Credentials. requestCode " +
                         "is %s", resultCode);
-                this.biometricAuth.getBiometricAuthListener().onError(
-                        ERROR_NO_DEVICE_CREDENTIAL,
+                this.biometricAuth.getBiometricAuthListener().onAuthenticationError(
+                        BiometricPrompt.ERROR_NO_DEVICE_CREDENTIAL,
                         "Fail to approve using device Credentials.");
```

Comment:
Can we have all the error strings as Enums or static strings somewhere centralized? t will help with alignement and localization

Reply:
UI-related string has been moved to the resource file, however, the current SDK design does not put all error log messages into the resource file.